indexStyle.css

```diff
@@ -0,0 +1,43 @@
+:root{

```

Comment:
You can move this in `public/assets/stylesheets/` and use only appropriate class names and ids.

Reply:
We can remove this file as well.